pkg/parse/parse.go

```diff
@@ -60,13 +60,6 @@ type branch struct {
 }
 
 func Parse() {
-	err, cb := o.Init()
-	if err != nil {
-		log.Error.Printf("could not validate command line options: %s", err)
-		cb()
-		os.Exit(1)
-	}
-
```

Comment:
Moved logging initialization up to the actual command line tools / build packages since log configuration is variable now.

If logging isn't configured when `Parse` is called, the default `stdout` loggers will be used.

I'd like to make an rc out of this PR once merged to make sure everything is working as intended for all of the wrappers